src/vue-components/popup.vue

```diff
@@ -674,10 +702,10 @@
         width: auto;
     }
 
-    .alpheios-popup__more-btn {
+    .auk .uk-button.alpheios-popup__more-btn {
         float: right;
         margin-bottom: 10px;
-        font-size: 0.675 * $alpheios-base-font-size !important;
+        font-size: 0.675 * $alpheios-base-font-size;
```

Comment:
Two points:  
1. I believe we were using `!important` here because we need to be sure the button font size is  not impacted by the base styles of whatever page the libraries are activated on. This is mostly an issue for the webextension though and now when the code is used in an embedded library so perhaps the best way to deal with that is via a skin in the webextension.  I will enter an issue to apply skins there.

2. I am not actually sure if the button text size should be affected by the user font-size preferences or not. it seems a little unexpected to me. I think I'd like to try it with having the font-size affect only the textual contents of the components but use a different button base font size for the buttons. What do you think about this @kirlat and @irina060981 ? Does it make sense?

Reply:
I have applied to buttons because it was mentioned at the text of the issue and I believe that all this resizing is used for making window bigger or smaller. For different person's visions. So all the text on the page become bigger or smaller.

I could create a variant with different sizes and paste screenshots here to make conclusions easier.